crypt.go

```diff
@@ -179,12 +183,39 @@ func (s *activateWithKeyDataState) tryActivateWithRecoveredKey(key DiskUnlockKey
 		return xerrors.Errorf("cannot activate volume: %w", err)
 	}
 
-	if err := keyring.AddKeyToUserKeyring(key, s.sourceDevicePath, keyringPurposeDiskUnlock, s.keyringPrefix); err != nil {
-		fmt.Fprintf(os.Stderr, "secboot: Cannot add key to user keyring: %v\n", err)
+	var firstDeviceStat *uint64
```

Comment:
nitpick: I would almost prefer a bool and uint64 to the pointer

Reply:
Done.